pom.xml

```diff
@@ -64,13 +62,23 @@
             <artifactId>bitbucket-git-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.atlassian.bitbucket.server</groupId>
+            <artifactId>bitbucket-git-common</artifactId>
```

Comment:
This allows you to use the `GitCommandExitHandler` directly, rather than hacking your way to it via Reflection